tests/langkit/metrics/test_pii.py

```diff
@@ -0,0 +1,82 @@
+# pyright: reportUnknownMemberType=none
+import pandas as pd
+
+import whylogs as why
+from langkit.core.metric import EvaluationConfig, EvaluationConfigBuilder
+from langkit.metrics.pii import prompt_response_presidio_pii_module
+from langkit.metrics.whylogs_compat import create_whylogs_udf_schema
+
+expected_metrics = [
+    "cardinality/est",
+    "cardinality/lower_1",
+    "cardinality/upper_1",
+    "counts/inf",
+    "counts/n",
+    "counts/nan",
+    "counts/null",
+    "distribution/max",
+    "distribution/mean",
+    "distribution/median",
+    "distribution/min",
+    "distribution/n",
+    "distribution/q_01",
+    "distribution/q_05",
+    "distribution/q_10",
+    "distribution/q_25",
+    "distribution/q_75",
+    "distribution/q_90",
+    "distribution/q_95",
+    "distribution/q_99",
+    "distribution/stddev",
+    "type",
+    "types/boolean",
+    "types/fractional",
+    "types/integral",
+    "types/object",
+    "types/string",
+    "types/tensor",
+    "ints/max",
+    "ints/min",
+]
+
+
+def _log(item: pd.DataFrame, conf: EvaluationConfig) -> pd.DataFrame:
+    schema = create_whylogs_udf_schema(conf)
+    return why.log(item, schema=schema).view().to_pandas()  # type: ignore
+
+
+def test_prompt_response_themes_module():
```

Comment:
nice. Lets add a version that just tests the prompt/response alone too with AAAIII